docs/crio.conf.5.md

```diff
@@ -141,6 +141,15 @@ the container runtime configuration.
 **blockio_config_file**=""
   Path to the blockio class configuration file for configuring the cgroup blockio controller.
 
+**cdi_spec_dirs**=[]
+  Directories to scan for CDI Spec files. The default list is:
```

Comment:
I'd like to be a bit more verbose here and link to a doc which describes what a CDI is.

Reply:
Reworded the text with slightly more context and a link to the main CDI site for more information.